openapi-spec/swagger.yaml

```diff
@@ -3304,6 +3382,15 @@ components:
                 Port for connecting to alert source
                 By default, set to 161
               example: 20162
+    SnmpConfigsRespSpec:
+      description: Response for all snmp alert source configuration.
+      type: object
+      properties:
+        snmp_configs:
+          type: array
+          description: the list of snmp configs
```

Comment:
Host example in spec (line no 3277) can be updated to show , seperated host list as it is new now

Reply:
Added